Del 2/main.py

```diff
@@ -7,20 +7,19 @@
 from app.point import Point
 from robot import Robot
 from robot.config import DRAWING_LEN
-from config import logger
```

Comment:
Var det slik at logging ikke fungerte? 🤔

Reply:
@BeatsuDev loggenren crasha på robotten. Det er ikke noe gæærnt med selve koden, det var bare et eller annet som kræsja på roboten da den kjørte.

For å ungå at det var to ting vi skulle debugge, så fjerna vi bare loggeren når vi skulle finne et setup som funket

Reply:
Og profileren kjører ikke på samme tid som loggeren

Reply:
> For å ungå at det var to ting vi skulle debugge
Hva mener du med dette?

Reply:
@BeatsuDev vi kan få loggeren til å fungere, men da tror jeg det er best å gjøre i en annen branch slik at vi ikke jobber med to ting samtidig